### PR TITLE
VIH-8687 update command execution timeout v2

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link (if applicable) ###
VIH-8687

### Change description ###
- updated graph api response to handle 403

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Anaia — APPROVED** [review]

File: UserApi/UserApi/Controllers/UserController.cs [hunk 1]
```diff
@@ -227,6 +227,10 @@ public async Task<IActionResult> DeleteUser([FromRoute]string username)
             {
                 return NotFound();
             }
+            catch (ForbiddenRequestToRemoveUserException)
+            {
+                return StatusCode((int)HttpStatusCode.Forbidden);
```
Comment (FarronF) [inline UserApi/UserApi/Controllers/UserController.cs:232]: Could this be 
```suggestion
                return Forbid();
```

I see some reasons why you might not want to but not sure if that applies here.
https://stackoverflow.com/questions/45095853/how-to-return-403-forbidden-response-as-iactionresult-in-asp-net-core

Comment (omio-ma) [reply UserApi/UserApi/Controllers/UserController.cs:232]: this was a suggestion from Anai and Arif, initially I was using Forbid, but in the unit tests we couldnt find a nice way to assert it (when you cast it as a ForbidResult Object it doesnt return anything to assert with). Returning the statuscode allowed us to actually assert status code properly and not just assert not null on the result of a cast

**Pull request closed without merging**